components/layout/header.tsx

```diff
@@ -15,7 +15,7 @@ export const Header = () => {
   return (
     <div className={"relative overflow-hidden bg-gradient-to-b"}>
       <LiveStreamBanner/>
-      <Container width="custom" size="custom" className="relative z-10 my-4 max-w-9xl py-0">
+      <Container width="custom" size="custom" padding="" className="relative z-10 my-4 max-w-9xl py-0">
```

Comment:
What is this change for?

Reply:
See changes to `Container` - default padding has been moved to the padding prop to allow it to be overridden if required